### PR TITLE
adds bamboo and  fixes sugarcane

### LINKED ISSUE #44170: Sugar Cane not present in seed dispenser
This is a pretty simple issue.  Sugar cane seeds are not loaded into the seed dispensers.  Some of the maps don't even have sugar cane crops in botany to get seeds from.  Should be a relatively minor fix.

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
add: bamboo which can be used to build punji sticks/ blowguns available as a sugarcane mutation or in exotic seed crate (instead of banana seeds)
fix: sugar cane is available at the mega seed vendor
/:cl:

closes: #44175
fixes:#44170

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
appveyor more like crapveyor

**Comment by ATH1909** [comment]
It's a bit of a shame that ashwalkers, the "tribal" role, won't really be able to make many blowguns without station or lifebringer assistance. Even if they got their hands on some bamboo seeds, they wouldn't be able to make any syringes to as ammo (their lack of easy access to syringes is really annoying when you loot the clown ruin, bring its rainbow extract back to your base that has multiple packets of blood that you can pour blood out of into containers, and then scream in frustration as you realize that you have no way to get the blood from a packet or container into the extract without station assistance).

And while we're at it, ashwalkers have no access to plasteel without raiding a ruin/the mining base/the station, so they can't make pestles on their own (they can make mortars, though, at least). I really don't see why the pestle needs to be made out of plasteel instead of, say, iron.

**Comment by fluffe9911** [comment]
use bamboo to make those fancy paper walls when

**Comment by nemvar** [comment]
Wasn't there this nerd on the last PR who said that mortar and pestle should both be the same material?

**Comment by boodaliboo** [comment]
i like bamboo pandas eat it but koalas dont

**Comment by boodaliboo** [comment]
also your pring a fix to sugarcane which isnt even merged might want to addthat fix to the pr

**Comment by OverlyGeneric** [comment]
space pandas when?

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1674,7 +1674,7 @@
 					/obj/item/seeds/liberty,
 					/obj/item/seeds/amanita,
 					/obj/item/seeds/reishi,
-					/obj/item/seeds/banana,
```
Comment (vuonojenmustaturska) [inline code/modules/cargo/packs.dm]: 🤔

Comment (Michael-Ashfield) [reply code/modules/cargo/packs.dm]: To be fair bananas are in the seed vendor already, and theres normally atleast 1 or 2 bananas lying around the station or in the corpse of the clown. There's no reason to have banana seeds in the seed crate aswell.

**Comment by Tlaltecuhtli** [comment]
diffbot blew up also i m not sure if i should give ashligs bamboos

**Comment by Tlaltecuhtli** [comment]
conflict resolved?

**Comment by optimumtact** [comment]
you still have a hidden change not listed in your changelog or removed from your PR

**Comment by Tlaltecuhtli** [comment]
changelog updated

**Comment by FantasticFwoosh** [comment]
I would be happier if the blowpipe could shoot pens, shotgun darts and other small/tiny objects as a mini p-cannon. Having bamboo open to use would simply leave it open to a more creative and invested person to add detail to later.  :eyes: 
> Blowdart shooting e-pen botanist meta

**Comment by optimumtact** [comment]
you still have a hidden change not listed in your changelog or removed from your PR

**Comment by Tlaltecuhtli** [comment]
updated @optimumtact

**Pull request merged**